### ISSUE TITLE
Functional: Docker install

### ISSUE THREAD

**Issue by joel72265** [lemnos/tt#41]
Thank you for the application. 

Please can you try to make a docker install of the program, that would be very good

**Comment by nnzv** [comment]
Here's a simple example  for you:

### Dockerfile

```dockerfile
# This stage sets up tools to build the project, clones it, and creates an executable named `tt`.
FROM golang:alpine AS build

ENV CGO_ENABLED=0 

WORKDIR /app

RUN apk add --no-cache git \
    && git clone --depth=1 https://github.com/lemnos/tt . \
    && go build -o bin/tt src/*.go

# Copies the `tt` executable and sets it as the command to run.
FROM alpine

COPY --from=build  /app/bin/tt /bin/tt

ENTRYPOINT ["tt"]
```

### How

```bash
# Build the Docker Image
docker build . -t dev:tt
# Run the Docker Container
docker run -it dev:tt
# To use `tt` with extra options
docker run -it dev:tt -h
```

**Comment by joel72265** [comment]
thank you for the steps :)

I will try to understand these step and follow

**Comment by joel72265** [comment]
hi, I am able to to run the following commands

**root@tt [ ~ ]**# curl -L https://github.com/lemnos/tt/releases/download/v0.4.2/tt-osx -o /usr/local/bin/tt && sudo chmod +x /usr/local/bin/tt
**root@tt [ ~ ]**# curl -o /usr/share/man/man1/tt.1.gz -L https://github.com/lemnos/tt/releases/download/v0.4.2/tt.1.gz
**root@tt [ ~ ]**# tt -quotes en

I am able to use the application on the local system using SSH but I would like to run it as a container (and port) so that I can use it from a Web Browser. If you have any steps that would be very useful. 

System: Photon OS

**Comment by nnzv** [comment]
This Go project is a TUI app. It can't turn into a web app easily. Seeing a web version of 'tt' would be intriguing. However, I think it should be a fork, not altering the original functionality. This goes beyond the project's scope.

**Comment by nz-v** [comment]
Seeing a web version of 'tt' would be intriguing. However, I think it should be a fork, not altering the original functionality. This goes beyond the project's scope.

**Comment by joel72265** [comment]
yes, even a fork version. A web based application **keeping the size minimal** would be great :)
